### PR TITLE
sys-libs/libblockdev: enable py3.12

### PR DESCRIPTION
**Pull request by parona-source**
Closes: https://bugs.gentoo.org/917028

### CODE REVIEW

File: dev-libs/libbytesize/libbytesize-2.10.ebuild [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		dev-util/gtk-doc
+		virtual/pkgconfig
+	)
+	test? (
+		dev-python/pocketlint[${PYTHON_USEDEP}]
+		dev-python/polib[${PYTHON_USEDEP}]
+	)
+"
+
+DOCS=( README.md )
```
Comment (mm1ke) [inline dev-libs/libbytesize/libbytesize-2.10.ebuild:44]: I think you can remove this.
`einstalldocs` already tries to install a certain set of files if available and `README.md` is included.

**Comment by thesamesam** [comment]
Please rebase / close if obsolete.

Marecki seems to have partially merged this without saying anything.

commit 1bfecae26244088337ceae3b36f9517637509924
Author: Marek Szuba <marecki@gentoo.org>
Date:   Tue Nov 21 09:37:17 2023 +0000

    dev-libs/libbytesize: drop 2.8

    Signed-off-by: Marek Szuba <marecki@gentoo.org>

commit 96989db795654cf4041a59bcd3f831d9fefe3527
Author: Alfred Wingate <parona@protonmail.com>
Date:   Thu Nov 9 02:23:38 2023 +0200

    dev-libs/libbytesize: add 2.10

    Signed-off-by: Alfred Wingate <parona@protonmail.com>
    Signed-off-by: Marek Szuba <marecki@gentoo.org>

**Comment by mkszuba** [comment]
Apologies for the delay, between work and my currently extremely flaky Internet connection I have only just had a chance to comment on this.

@parona-source : I have merged the commits pertaining to pocketlint and libbytesize but unfortunately the libblockdev has got issues:
* you have in a single commit enable py3.12 support, updated+unrestricted the test phase, and added a patch fixing distro identification on OpenRC systems. One functional change per commit, please!
* all the tests fail with errors, at least when emerged on a clean stage3 system anyway. I have already added two missing test dependencies but either this isn't enough or there are some other issues present. Please review.

**Comment by parona-source** [comment]
```
======================================================================
ERROR: test_force_plugin (library_test.PluginsTestCase.test_force_plugin)
Verify that forcing plugin to be used works as expected
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/tmp/portage/sys-libs/libblockdev-3.0.4/work/libblockdev-3.0.4/tests/library_test.py", line 243, in test_force_plugin
    self.assertTrue(BlockDev.reinit([ps], True, None))
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/tmp/portage/sys-libs/libblockdev-3.0.4/work/libblockdev-3.0.4/src/python/gi/overrides/BlockDev.py", line 144, in reinit
    return _reinit(require_plugins, reload, log_func)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
gi.repository.GLib.GError: g-bd-init-error-quark: Failed to load plugins (1)
```

Enabling lvm handled this for me, but it may be a case where one of many would suit better. Haven't looked further

**Pull request closed without merging**